8-lambda/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM amazonlinux:2017.03.1.20170812
```

Comment:
I wonder if we should use `latest` here so this does get stale?

Reply:
Sadly, Lambda [lags behind `latest` a good bit](https://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html).  They've had [three releases](https://hub.docker.com/_/amazonlinux/) since the version used by Lambda.